client/src/components/Poster.spec.js

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react'
+import Poster from './Poster'
+
+describe('Poster', () => {
+  it('renders', () => {
+    render(<Poster />)
+
+    const poster = screen.getByRole('img')
+    expect(poster).toBeInTheDocument()
+  })
+
+  it('has the correct image path', () => {
+    const imagePath = 'https://via.placeholder.com/300x450?text=POSTER'
```

Comment:
To make your tests independent of external ressources, you could use a local link.

Reply:
Does not even need to be a working link here.